src/components/WeekTabContent/WeekTabContent.jsx

```diff
@@ -0,0 +1,128 @@
+        if (currentWeekTasks) return true;
+        else return false;
+    }
+
+    return(
+        <>
+        <WeekTabContentWrapper isTasksAvailable = {isTasksAvailable()}>
+            <WeekInfoHeader>
+                <CurrentDateData>
+                    <WeekData>
+                        {weekDataString}
+                    </WeekData>
+                    <DayDataSet>
+                        <MyTasksText>
+                            My tasks:
+                        </MyTasksText>
+                        <DayData>
+                            {selectedDay.toUpperCase()}, {selectedDate.getDate()}-{selectedDate.getMonth()+1}-{selectedDate.getFullYear()}
+                        </DayData>
+                    </DayDataSet>
+                </CurrentDateData>
+                <WeekTasksData>
+                    <WeekTasksTextElemSet>
+                        <ElemSetTitle>
+                            Points gained this week:
+                        </ElemSetTitle>
+                        <ElemSetValue>
+                            {currentWeekTasks && currentWeekTasks.rewardsGained || 0}
+                        </ElemSetValue>
+                    </WeekTasksTextElemSet>
+                    <WeekTasksTextElemSet>
+                        <ElemSetTitle>
+                            Points planned for this week:
+                        </ElemSetTitle>
+                        <ElemSetValue>
+                            {currentWeekTasks && currentWeekTasks.rewardsPlanned || 0} 
+                        </ElemSetValue>
+                    </WeekTasksTextElemSet>
+                    <ProgressBarSet>
+                        <ProgressBarInfo>
+                            <InfoTasksDone>
+                                {currentWeekTasks && currentWeekTasks.rewardsGained || 0}&nbsp;
+                            </InfoTasksDone>
+                            <InfoTasksPlanned>
+                                / {currentWeekTasks && currentWeekTasks.rewardsPlanned || 0} 
+                            </InfoTasksPlanned>
+                        </ProgressBarInfo>
+                        <ProgressBarBackground>
+                            <ProgressBarFiller width = {getProgressBarFillPercent()}>
+                            </ProgressBarFiller>
+                        </ProgressBarBackground>
+                    </ProgressBarSet>
+                </WeekTasksData>
+            </WeekInfoHeader>
+            <CurrentTasksArea>
+                {
+                    currentWeekTasks && <span>tasks here</span>
+                }
+            </CurrentTasksArea>
+            {
+                !currentWeekTasks && 
+                <NoTasksScreen>
+                    <NoTasksScreenText>
+                        No tasks for this day
+                    </NoTasksScreenText>
+                    <NoTasksScreenPlanBtn>
+                        Plan tasks
+                    </NoTasksScreenPlanBtn>
```

Comment:
Clicking on the "plan" button should do some action, should it? I think it could be navigation to planning page